src/app/fetcher/fullDomFetcher.js

```diff
@@ -8,7 +8,7 @@ import { FetchDocumentError } from './errors.js';
 puppeteer.use(StealthPlugin());
 
 const PUPPETEER_TIMEOUT = 30 * 1000; // 30 seconds in ms
-
+const WAIT_FOR_ELEMENTS_TIMEOUT = config.has('fetcher.waitForElementsTimeout') ? config.get('fetcher.waitForElementsTimeout') : 10 * 1000; // 10 seconds in ms
```

Comment:
Why do we set a default value here instead of relying on defaults being set at config level?

Reply:
Yes, I knew that this point would be discussed :).
I did that to avoid the consumer of the command validate outside this repo, like the repo `service-all`, to have to define a config. Let's discuss about that.